Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Magazines/light_rifle.yml

```
@@ -83,8 +86,9 @@
 
 - type: entity
   id: MagazineLightRiflePractice
-  name: "magazine (.30 rifle practice)"
+  name: "magazine (.308 rifle practice)"
   parent: BaseMagazineLightRifle
+  description: Standard AKMS magazine holding 30 rounds. This version contains practice ammo with blue rubber tips.
```

We often use "supposed to contain" for things that can be potentially refilled. Canisters for example use this.

Also recognize that rubber rounds != practice rounds.

Suggestion:
```suggestion
  description: Standard AKMS magazine holding 30 rounds. This version is supposed to contain practice ammo.
```